src/getting-started.md

```diff
@@ -41,6 +41,63 @@ A screenshot of the Flix Visual Studio Code extension in action:
 
 ![Visual Studio Code1](images/vscode1.png)
 
+### Using Flix from Neovim
+
+Flix can also be used from Neovim. Follow these steps to make it work:
+- Install neovim
+- Add plugin `lspconfig` to your neovim through your favorite neovim plugin manager or run this command if you are not using any plugin manager(assuming you have `~/.config/nvim` as your neovim config directory):
```

Comment:
Add a zeroth step to check that nvim is installed and atleast version 0.9x (it says upon start up).